.github/workflows/ci.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       matrix:
         # Right now macos-13 is amd64 while macos-14 is aarch64 (M1)
         os: [ubuntu-latest, macos-13, macos-14, windows-latest]
+        pkl_version: [0.26.0, 0.26.1, 0.26.2, 0.26.3]
```

Comment:
Do we really need/want a matrix test for each and every single pkl version? 🤔
I have the feeling that is a bit overkill..

Reply:
Haha probably not, I added it in this (still in draft!) PR to make sure the checking of version strings was working (properly interpolated, not accidentally comparing to a hard-coded value, say).

I will probably leave in the matrix but with only one array element - in case Pkl changes their download mechanism in the future, and we want to test new behaviour for downloading from both old methods and new.